django_loci/geo/geocode.py

```diff
@@ -0,0 +1,41 @@
+from django.http import JsonResponse
+from django.views import View
+from django.utils.module_loading import import_string
+
+from .settings import DJANGO_LOCI_GEOCODER, DJANGO_LOCI_GEOCODE_API_KEY, DJANGO_LOCI_GEOCODE_DELAY, DJANGO_LOCI_GEOCODE_RETRIES
+
+from geopy.extra.rate_limiter import RateLimiter
+
+
+class GeoCodeView(View):
+    geocoder = import_string('geopy.geocoders.' + DJANGO_LOCI_GEOCODER)
```

Comment:
learn to use `format`: `'geopy.geocoders.{}'.format(DJANGO_LOCI_GEOCODER)` which is more pythonic